### ISSUE TITLE
[Feature Request] Customizable Swipe Up Gesture

### ISSUE THREAD

**Issue by onnyyonn** [MM2-0/Kvaesitso#993]
Currently, the swipe up gesture opens the widgets, and it's not customizable unlike swipe down/left/right. Please allow binding another gesture to widgets and free up "swiipe up" so that it can be used for other tasks. Opening the app drawer is a common use case for the swipe up gesture.

**Comment by blubzie** [comment]
yeah and for me its nothing against the app this decision. I am migrating from another launcher with swipe up to search and thats just muscle memory now. It seems like such a small thing to make this customizeable, I really dont understand why not just do it

**Comment by loxoron218** [comment]
Any news on this?

**Comment by gnu1dea** [comment]
Duplicate of #1164
